### PR TITLE
Fixed the problem with the authentication not displaying errors.

### PR DESCRIPTION
**Pull request by brec9824**
Little fix to allow $errors var to now contain authentication errors. $errors was getting erased by double calls to web middle-ware which is now fixed.

### CODE REVIEW

File: app/Http/routes.php [hunk 1]
```diff
@@ -12,27 +12,19 @@
...
+Route::get('/', 'PagesController@home');
+Route::get('about', 'PagesController@about');
+Route::get('projectfinder', 'PagesController@projectfinder');
 
-    //Route::get('register', 'PagesController@register');
-    //Route::get('login', 'PagesController@login');
-    //Route::get('reset', 'PagesContorller@reset');
-});
\ No newline at end of file
+//Route::get('register', 'PagesController@register');
```
Comment (rickbau5) [inline app/Http/routes.php]: Is this commented section necessary?

Comment (brec9824) [inline app/Http/routes.php]: Oops no it is not. That was the old routing before I used Route::auth(); to control the authentication routing.

**Comment by rickbau5** [comment]
Looks good :+1: merging!

**Comment by rickbau5** [comment]
Others: To update your branch do the following:

```
git pull
git stash     //this "stashes" any changes you have to your branch, without commiting them
git rebase origin master   //this updates your branch to master, then reapplies commits that you've made back on to it
git stash apply    //this reapplies the stashed changes, bringing your branch back to where it was
```

If you get any errors or merge conflicts, let me know and I can help you fix.

**Pull request merged**